query-engine/core/src/executor/interpreting_executor.rs

```diff
@@ -52,13 +52,16 @@ where
         if let Some(tx_id) = tx_id {
             self.itx_manager.execute(&tx_id, operation, trace_id).await
         } else {
-            execute_single_self_contained(
-                &self.connector,
-                query_schema,
-                operation,
-                trace_id,
-                self.force_transactions,
-            )
+            super::with_request_now(async move {
```

Comment:
I know this is the best way for now, I'm a bit worried though somebody adds a new operation, forgets this and we find it out way too late.

Reply:
As noted above, it will panic very early and unambiguously. One test with an `@default(now())` that uses the new method is coverage enough. We both agree it's not the ideal solution, but I'm not too worried about this specific problem.